exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillOptiq.java

```diff
@@ -22,7 +22,7 @@
 import java.util.LinkedList;
 import java.util.List;
 
-import org.apache.drill.shaded.guava.com.google.common.base.Preconditions;
+import org.apache.calcite.rel.type.RelDataType;
```

Comment:
Optiq is an old name of Calcite. Does it make sense to rename this class?

Reply:
It probably does..can you create a JIRA for the renaming since it is independent of this PR.